src/swarm/neo/connection/ConnectionBase.d

```diff
@@ -972,26 +971,26 @@ abstract class ConnectionBase: ISelectClient
 
```

Comment:
Thanks. Will add it. The thing is we were not checking the return value of the setsock anyway.